### PR TITLE
Add note about python3-venv package on Debian/Ubuntu

### PR DESCRIPTION
**Pull request by yakky**

### CODE REVIEW

**Comment by simobasso** [comment]
+1! debian testing and sid are "affected".

File: en/django_installation/instructions.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ It will look like this:
 
 `myvenv` is the name of your `virtualenv`. You can use any other name, but stick to lowercase and use no spaces. It is also good idea to keep the name short as you'll be referencing it a lot!
 
+> __NOTE:__ On some versionsof Debian/Ubuntu you may receive the following error
```
Comment (olasitarska) [inline en/django_installation/instructions.md]: `versionsof` -> `versions of`

Comment (yakky) [inline en/django_installation/instructions.md]: Fixed

**Comment by olasitarska** [comment]
Thanks @yakky! :dog2:

**Pull request merged**